diff --git a/cmd/tendermint/commands/root.go b/cmd/tendermint/commands/root.go
index 6d79f75c..ba1e9977 100644
--- a/cmd/tendermint/commands/root.go
+++ b/cmd/tendermint/commands/root.go
@@ -4,6 +4,7 @@ import (
 	"fmt"
 	"os"
 
+	"github.com/oneiro-ndev/o11y/pkg/honeycomb"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 
@@ -13,9 +14,32 @@ import (
 	"github.com/tendermint/tendermint/libs/log"
 )
 
+func initlog() log.Logger {
+	var l log.Logger
+	if logwriter, err := honeycomb.NewWriter(); err != nil {
+		// if honeycomb fails, just use the standard tm logger
+		l = log.NewTMLogger(log.NewSyncWriter(os.Stdout))
+	} else {
+		l = log.NewTMJSONLogger(log.NewSyncWriter(logwriter))
+		chain := os.Getenv("CHAIN")
+		if chain == "" {
+			chain = config.ProxyApp
+		}
+		l = l.With("bin", "tendermint-" + chain)
+		nodeID := os.Getenv("NODE_ID")
+		if nodeID == "" {
+			nodeID = config.Moniker
+		}
+		l = l.With("node_id", nodeID)
+		l.With("hives", 2).Info("honeycomb initialized")
+	}
+	return l
+}
+
 var (
 	config = cfg.DefaultConfig()
-	logger = log.NewTMLogger(log.NewSyncWriter(os.Stdout))
+	// this is the default logger in case we can't initialize honeycomb
+	logger = initlog()
 )
 
 func init() {
